ui/jetbrains/src/main/java/com/neueda/jetbrains/plugin/graphdb/jetbrains/ui/datasource/metadata/dto/ContextMenu.java

```diff
@@ -0,0 +1,35 @@
+package com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.datasource.metadata.dto;
+
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.component.datasource.state.DataSourceApi;
+import com.neueda.jetbrains.plugin.graphdb.jetbrains.ui.datasource.tree.NodeType;
+
+public class ContextMenu {
+
+    private NodeType metadataType;
+    private DataSourceApi dataSourceApi;
+    private String data;
+
+    public ContextMenu(NodeType metadataType, DataSourceApi dataSourceApi) {
+        this.metadataType = metadataType;
+        this.dataSourceApi = dataSourceApi;
+    }
+
+    public ContextMenu(NodeType metadataType, DataSourceApi dataSourceApi, String data) {
+        this.metadataType = metadataType;
+        this.dataSourceApi = dataSourceApi;
+        this.data = data;
+    }
+
+    public NodeType getMetadataType() {
+        return metadataType;
+    }
+
+    public DataSourceApi getDataSourceApi() {
+        return dataSourceApi;
+    }
+
+    public String getData() {
```

Comment:
There is possibility that data will be null. Either mark with annotation `@Nullable` (from intellij platform) or add `Optional`.